coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -3324,7 +3324,7 @@ def _internal_op_tensor_inplace_fill(context, node):
     begin, end, stride, begin_mask, end_mask, squeeze_mask = _get_slice_params(
         context, data, node.inputs[2:]
     )
-    if begin.val is None or end.val is None:
+    if begin.val is None or end.val is None or any_symbolic(data.shape):
```

Comment:
I found `_internal_op_tensor_inplace_fill` does not work with flexible shape data.
`update_values = _np.full(fill_shape, fill_scalar.val)` does not work.